libraries/chain/include/scorum/chain/services/registration_committee.hpp

```diff
@@ -14,14 +14,13 @@ namespace chain {
 
 struct registration_committee_service_i : public scorum::protocol::registration_committee_i
 {
-    using registration_committee_member_refs_type = std::vector<registration_committee_member_object::cref_type>;
+    using member_object_cref_type = std::vector<registration_committee_member_object::cref_type>;
```

Comment:
because it is registration_committee_member_object reference type.

Reply:
It is no good, because you have to include: 
#include <scorum/chain/schema/registration_objects.hpp> not predefined class registration_committee_member_object